### DISCUSSION TITLE
What are the best ways to remove duplicated publications in my dataset?

### DISCUSSION THREAD

**Discussion by J535D165** [asreview/asreview#529] [Q&A] [answered]
Duplicates are annoying during screening and might affect the performance of ASReview (negatively). What are the best ways  and tools to remove duplicates from my dataset?

**Comment by SagevdBrand** [comment, marked as answer]
Indeed ASReview works suboptimally when there are duplicates! Using reference managers, you can deduplicate your dataset quite easily. You can check these references for how to remove the duplicates:
- Bramer, W. M., Giustini, D., de Jonge, G. B., Holland, L., & Bekhuis, T. (2016). De-duplication of database search results for systematic reviews in EndNote. Journal of the Medical Library Association : JMLA, 104(3), 240–243. [link](https://doi.org/10.3163/1536-5050.104.3.014)
- Third-party manuals for removing duplicates in Zotero, Mendeley, and RefWorks, can found in the folder 4. Deduplication by Staaks (2020).
- Staaks, J. (2020, October 15). Systematic Review Search Support. [link](https://doi.org/10.17605/OSF.IO/49T8X)
